Builds/Test.py

```diff
@@ -129,14 +169,76 @@ def powerset(iterable):
     help='Reduce output where possible (unit tests)',
 )
 
+# Scons and CMake parameters are too different to run
+# both side-by-side
+pgroup = parser.add_mutually_exclusive_group()
+
+pgroup.add_argument(
+    '--cmake', '-m',
+    action='store_true',
+    help='Build using CMake.',
+)
+
+pgroup.add_argument(
+    '--scons',
+    action='store_true',
+    help='Build using Scons. Default behavior.')
```

Comment:
Right. Eventually, cmake will be the default, so I'm putting this in place now for anybody who cares enough to stick with scons when it changes.